src/content/docs/apm/agents/net-agent/other-installation/understanding-net-agent-environment-variables.mdx

```diff
@@ -0,0 +1,217 @@
+---
+title: Understanding required .NET agent environment variables
+tags:
+  - Agents
+  - NET agent
+  - Other installation
+redirects:
+---
+
+Our .NET agent uses environment variables to attach the profiler to your application, which allows the .NET agent and profiler to find required assets. If you're interested in diving deeper into how environment variables are used and which ones.
```

Comment:
You're right, actually let's build out these first few sentences. How does something like this look?

```suggestion
Our .NET agent is a .NET CLR profiler and relies on your app's environment variables to connect your .NET app with New Relic. To learn more about which environment variables the .NET agent uses, see the sections below. 
```